core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1016,6 +1016,15 @@ package object config {
       .booleanConf
       .createWithDefault(true)
 
+  private[spark] val MAP_STATUS_COMPRESSION_CODEC =
+    ConfigBuilder("spark.shuffle.mapStatus.compression.codec")
+      .doc("The codec used to compress MapStatus, which is generated by ShuffleMapTask. " +
+        "By default, Spark provides four codecs: lz4, lzf, snappy, and zstd. You can also " +
+        "use fully qualified class names to specify the codec. If this is not given, " +
+        "spark.io.compression.codec will be used.")
```

Comment:
we can remove this sentence if we don't fallback to spark.io.compression.codec

Reply:
oops